lib/suggest.js

```diff
@@ -77,6 +77,8 @@ function suggest(value) {
 
   while (++index < length) {
     character = value.charAt(index)
+    before = value.slice(0, index)
+    after = value.slice(index + 1)
```

Comment:
Here we `slice` the word at the index of the `character` in the input word. From my perspective, this more accurately simulates keyboard errors than the current approach.